src/windowmanagement/data/MenuItems.tsx

```diff
@@ -138,7 +142,7 @@ export const menuItemsData: Record<string, MenuItem> = {
     content: <ExoplanetsPanel />,
     renderIcon: (size) => <ExoplanetIcon size={size} />,
     preferredPosition: 'right',
-    defaultVisible: true
+    defaultVisible: false
```

Comment:
I'm confused. It seems like this is still the setting we listen to, not the setting in the `default_ui_panels.json` file?

Reply:
Conclusion form discussion - change this back to true
```suggestion
    defaultVisible: true
```